tools/patch-ssl

```diff
@@ -34,25 +34,15 @@ fi
 # Construct "self-extracting" cert patch script with embedded cert
 
 cat >patch-cert  <<EOF
-cat >tls-ca-bundle.pem <<END_CERT
+cd /etc/ssl/certs
+cat >stsci-tls-ca-bundle.pem <<END_CERT
 EOF
 
 cat ${CERT_FILE} >>patch-cert
 
 cat >>patch-cert  <<EOF
 END_CERT
```

Comment:
Can you help me understand what's going on with these nested EOFs please?

Reply:
Yeah.   A would-be-nice feature missing from the current patch system is the ability to include auxiliary files in addition to the patch script.   patch-ssl generates the patch-cert script which will be run inside the container from a little boilerplate and the new cert file.     Since there is no way to include aux files,  the cert is just inlined in the patch-cert script.    EOF only exists in patch-ssl.   END_CERT is written into patch-cert.   If you squint,  you can see there are 3 statements in patch-ssl which build up patch-cert something like:  boilerplate + new-cert + boilerplate.

Reply:
>  EOF only exists in patch-ssl. END_CERT is written into patch-cert

OH, ok.  Thanks :)  That was hurting my brain.